src/Fabulous.AST/Core/Builders.fs

```diff
@@ -235,6 +229,9 @@ type CollectionBuilder<'marker, 'itemMarker> =
         member inline _.Yield(widget: WidgetBuilder<'itemMarker>) : CollectionContent =
             { Widgets = MutStackArray1.One(widget.Compile()) }
 
+        member inline _.YieldFrom(widget: WidgetBuilder<'itemMarker>) : CollectionContent =
+            { Widgets = MutStackArray1.One(widget.Compile()) }
```

Comment:
`yield!` should take a list of widgets, rather than a single widget otherwise you're just duplicating the `yield` on top.

Something like:

```fs
let myField = [
    Field("Hello")
    Field("World")
    Fieldl("!")
]

Record(...) {
    yield! myFields
}
```

which is the equivalent of doing

```fs
let myField = [
    Field("Hello")
    Field("World")
    Fieldl("!")
]

Record(...) {
   for field in myFields do
       yield field
}
```